modules/cloud-create-instance-steps.adoc

```diff
@@ -20,9 +20,11 @@ To create an *ACS instance*:
 . From the navigation menu, select *Advanced Cluster Security* -> *ACS Instances*.
 . Select *Create ACS instance* and enter information into the displayed fields or select the appropriate option from the drop-down list:
 * *Name*: Enter the name of your *ACS instance*. An *ACS instance* contains the {product-title-short} Central component, also referred to as "Central," which includes the {product-title-managed-short} management interface and services that are configured and managed by Red Hat. You manage your secured clusters that communicate with Central. You can connect multiple secured clusters to one instance.
-* *Cloud provider*: The cloud provider where Central is located. Select *AWS*. 
-* *Cloud region*: The region for your cloud provider where Central is located. Select *AWS*.
-* *Availability zones*: Use the default value.
+* *Cloud provider*: The cloud provider where Central is located. Select *AWS*.
+* *Cloud region*: The region for your cloud provider where Central is located. Select one of the following:
```

Comment:
```suggestion
* *Cloud region*: The region for your cloud provider where Central is located. Select one of the following regions:
```

Reply:
From IBM Style:

> **following** _adjective_, _noun_
To introduce a list, procedure, or subsequent items, always use _following_ as an adjective before a noun. For example, do not write “Complete the following” or ”The functions include the following”. Instead, write “Complete the following steps” or “The adapter has the following functions”.